lib.spec.js

```diff
@@ -60,10 +60,7 @@ describe('Итераторы', () => {
         const invitedFriends = [];
 
         while (!maleIterator.done() && !femaleIterator.done()) {
-            invitedFriends.push([
-                maleIterator.next(),
-                femaleIterator.next()
-            ]);
+            invitedFriends.push([maleIterator.next(), femaleIterator.next()]);
```

Comment:
![one-code-style-to-rule-them-all](https://user-images.githubusercontent.com/18099714/49010530-60b9a380-f195-11e8-84ad-ff77a6ac090d.jpg)